README.md

```diff
@@ -2,68 +2,50 @@
 
 This repository contains the merchant-component of the Pricewars-simulation. The merchant represents a vendor trying to sell their products at the highest possible profit at the marketplace. As such, they also represent a certain pricing strategy that is to be tested against other pricing strategies. Such a pricing strategy can be either very simple and rule-driven, such as "always be 10 cents cheaper than your competitors" or data-driven and use machine learning strategies.
 
-**Have a look at the IPython Notebooks [for the Marketplace and Product API](merchant_sdk/samples/Handling%20products%20and%20offers.ipynb) and [Kafka and Data API](merchant_sdk/samples/Working%20with%20Kafka%20data.ipynb)!**
-
-This repository contains multiple sample merchants that each implement one of the following strategies:
+There is an example merchant available (in `merchant.py`) that implements the following strategies:
 
 * Cheapest
-* Second Cheapest
-* Random Third
-  * randomly chooses either the first, second or third position in the price ranking
-* Two Bound 
+  * always undercuts the competitors
+* Two Bound
   * has a minimum and maximum profit margin
   * keeps decreasing the price to maintain the 1st position in the price rank until the minimum bound is reached
   * then set the price back to the maximum margin
-* Fix Price
-* Maximize Highest Expected Profit
-  * i.e. a data-driven, machine learning merchant using logistic regression
 
-Moreover, the repository contains an SDK to easily build arbitrary merchant behaviors yourself.
+There are three ways to build your own merchant:
+* Use the example merchant and make changes to it (`merchant.py`)
+* [Subclass the merchant base class and extend it](docs/Building%20a%20merchant%20using%20PricewarsMerchant.ipynb)
```

Comment:
Is this notebook still fully up to date?

Reply:
This notebook is newly created in this PR